site-search/site-search_test.go

```diff
@@ -323,8 +323,6 @@ func TestClient_GetReleases(t *testing.T) {
 					ReleaseDate: time.Now().AddDate(0, 0, -10).UTC().Format(time.RFC3339),
 					Published:   true,
```

Comment:
could you add these fields to your test pls:
Census
 Keywords      
 ProvisionalDate 
 Language

Reply:
Okay that's done